src/core/AnimatedEvent.js

```diff
@@ -1,19 +1,28 @@
-import { findNodeHandle } from 'react-native';
+import { findNodeHandle, Platform } from 'react-native';
 import ReanimatedModule from '../ReanimatedModule';
 
 import AnimatedNode from './AnimatedNode';
 import AnimatedValue from './AnimatedValue';
+import AnimatedAlways from './AnimatedAlways';
 
 import invariant from 'fbjs/lib/invariant';
+import createAandroidProxyPolyfill from './AndroidProxyEventPolyfill';
```

Comment:
can we rename that file to match the method we expect it to provide? There is a typo here as well -> "Aandroid"

I'd suggest the following name: createEventObjectProxyPolyfill